src/SUMMARY.md

```diff
@@ -292,7 +292,7 @@
   - [Async Traits](async/pitfalls/async-traits.md)
 - [Exercises](exercises/concurrency/afternoon.md)
   - [Dining Philosophers](exercises/concurrency/dining-philosophers-async.md)
-  - [Elevator Operations](exercises/concurrency/elevator.md)
+  - [Broadcast Chat Application](exercises/concurrency/chat-app.md)
```

Comment:
Please add a redirect for this — and I suppose the `elevator.md` file should be deleted now?

Reply:
Could you just remove the directory now? I would like `src/` to have no unused files lying around.

Reply:
Removed all the `elevator` exercise files.

Reply:
Thanks!